dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/net/info/CallbackNetworkInfoProvider.kt

```diff
@@ -60,19 +61,31 @@ internal class CallbackNetworkInfoProvider :
             }
         } catch (e: SecurityException) {
             // RUMM-852 On some devices we get a SecurityException with message
-            // "package does not belong to 10411"
+            // "package does not belong to xxxx"
+            devLogger.e(ERROR_REGISTER, e)
+            networkInfo = NetworkInfo(NetworkInfo.Connectivity.NETWORK_OTHER)
+        } catch (e: RuntimeException) {
+            // RUMM-918 in some cases the device throws a IllegalArgumentException on register
```

Comment:
are we causing this or is just related with the fact that on the client side they're using too many callbacks ?

Reply:
No idea, we can't reproduce this